packages/flutter_tools/test/host_cross_arch.shard/ios_content_validation_test.dart

```diff
@@ -141,7 +146,18 @@ void main() {
           printOnFailure(buildResult.stderr.toString());
           expect(buildResult.exitCode, 0);
 
-          expect(outputPluginFrameworkBinary, exists);
+          // Plugins are built either as a static library (SwiftPM's default)
+          // or as a dynamic library (CocoaPods's default).
+          // If built as a dynamic library, the plugin will have a .framework.
+          // If built as static library, the plugin's symbols will be in the
+          // Runner binary.
+          final bool helloDynamic = outputPluginFrameworkBinary.existsSync();
+          final bool helloStatic = AppleTestUtils
+            .getExportedSymbols(outputRunnerBinary.path)
```

Comment:
`AppleTestUtils.getExportedSymbols` is `flutter_tools`'s helper for `nm`: https://github.com/flutter/flutter/blob/91813060c0525e4015c1c11bfd38e95165f19d3e/packages/flutter_tools/test/integration.shard/test_utils.dart#L114-L129